core/primitives/Cargo.toml

```
@@ -39,8 +39,10 @@ near-vm-errors = { path = "../../runtime/near-vm-errors" }
 near-rpc-error-macro = { path = "../../tools/rpctypegen/macro" }
 
 [features]
-default = ["jemallocator"]
+default = ["jemallocator", "protocol_feature_lower_storage_cost"]
```

I think we should add `stable_protocol_features` here instead.
```suggestion
default = ["jemallocator", "stable_protocol_features"]
```